src/ol_concourse/pipelines/open_edx/edx_platform_v2/grouped_docker_packer_pulumi_pipeline.py

```diff
@@ -0,0 +1,305 @@
+        # TODO MD 20230512 Fix to main branch once testing completed
+        branch="md/edxapp_docker_migration",
+        paths=[
+            "dockerfiles/openedx-edxapp/",
+            "src/ol_concourse/pipelines/open_edx/edx_platform_v2",
+        ],
+    )
+
+    container_fragments = []
+    packer_fragments = []
+    pulumi_fragments = []
+    group_configs = []
+    for release_name in releases:
+        job_names = []
+        for deployment in filter_deployments_by_release(release_name):
+            theme_git_resources = []
+            theme_get_steps = []
+
+            theme = fetch_application_version(
+                release_name, deployment.deployment_name, OpenEdxApplication.theme
+            )
+            theme_git_resource = git_repo(
+                name=Identifier(f"{release_name}-{deployment.deployment_name}-theme"),
+                uri=theme.git_origin,
+                branch=theme.release_branch,
+            )
+            theme_git_resources.append(theme_git_resource)
+            theme_get_steps.append(GetStep(get=theme_git_resource.name, trigger=False))
+
+            edx_platform_git_resources = []
+            edx_platform_get_steps = []
+            edx_platform = fetch_application_version(
+                release_name, deployment.deployment_name, OpenEdxApplication.edxapp
+            )
+            edx_platform_git_resource = git_repo(
+                name=Identifier(
+                    f"{release_name}-{deployment.deployment_name}-edx-platform"
+                ),
+                uri=edx_platform.git_origin,
+                branch=edx_platform.release_branch,
+            )
+            edx_platform_git_resources.append(edx_platform_git_resource)
+            edx_platform_get_steps.append(
+                GetStep(get=edx_platform_git_resource.name, trigger=False)
+            )
+
+            edx_ami_code = git_repo(
+                name=Identifier(f"edxapp-custom-image-{deployment.deployment_name}"),
+                uri="https://github.com/mitodl/ol-infrastructure",
+                # TODO MD 20230512 Fix to main branch once testing completed
+                branch="md/edxapp_docker_migration",
+                paths=[
+                    "src/bridge/settings/openedx/",
+                    "src/bilder/components/",
+                    "src/bilder/images/edxapp_v2/deploy.py",
+                    "src/bilder/images/edxapp_v2/group_data/",
+                    "src/bilder/images/edxapp_v2/files/",
+                    "src/bilder/images/edxapp_v2/templates/vector/",
+                    f"src/bilder/images/edxapp_v2/templates/edxapp/{deployment.deployment_name}/",  # noqa: E501
+                    "src/bilder/images/edxapp_v2/custom_install.pkr.hcl",
+                    f"src/bilder/images/edxapp_v2/packer_vars/{deployment.deployment_name}.pkrvars.hcl",  # noqa: E501
+                    f"src/bilder/images/edxapp_v2/packer_vars/{release_name}.pkrvars.hcl",  # noqa: E501
```

Comment:
We can probably just make that `src/bilder/images/edxapp_v2/`

Reply:
I was thinking to reduce the number of triggered builds. No sense building everything if we've only changed the settings file for mitx-staging or w/e. But I'm fine to go either way.